### DISCUSSION TITLE
Zavala 2.7

### DISCUSSION THREAD

**Discussion by vincode-io** [vincode-io/Zavala#245] [General]
Hot on the heels of 2.6, 2.7 is now live and in production. This is the version that supports Nested Tags.

Nested Tags are a nice way to add more complex ways to organize Outlines. See the Zavala Help for more information on how to use Nested Tags. The great thing about Nested Tags is that if you don't need their complexity, it just isn't there. You can still use Tags (or not use Tags) just as you always have.

Another bigger thing I worked on was interoperability with other applications. Zavala now supports pasting and dropping of Markdown unordered lists. This means that you can drag from OmniOutliner to Zavala and Zavala will automatically format the dragged Rows correctly. You can also do this from a list in macOS's Notes app. Actually any app that supports generating a Markdown unordered list will work.

This might be the last of the 2.x series for Zavala. Minor bug fix releases will still happen as needed of course. I really feel like Zavala is a lot more stable and feature complete than it was a year ago when 2.0 was released. This gives me some breathing room to update Zavala internally to Apple's latest API's and technologies. Zavala 3.0 will probably work towards supporting the Apple Intelligence stuff announced at WWDC this year.

**Comment by jsismail** [comment]
Which version of MacOS did you test the Notes interoperability with?  Cutting and pasting a list from Notes into Zavala doesn't result in a Zavala outline for me.  Rather, it results in a single multiline row.

**Reply by vincode-io** [reply]
If you have your cursor in a text field (Edit Mode), Zavala will just paste the text as you found out. You need to be in Outline Mode (hit ESC) to paste as an Outline.

**Reply by jsismail** [reply]
Ah yes this works for me. Nice!

I'm starting to understand better where you're drawing the line with this type of support, and it seems to be Markdown. I think that's reasonable. Now I'm wondering if there's an easy way to convert a lists/outlines from Notes built using cmd-[ into markdown.
